src/main/java/ca/on/oicr/pineryreports/reports/impl/BisqueProjectsStatusReport.java

```diff
@@ -261,7 +261,7 @@ protected void collectData(PineryClient pinery) throws HttpResponseException, IO
     seqdLibsByCategory.get(COUNT_CATEGORY_NON_ILL_SEQD).put("All", new HashSet<>());
 
     
-    // track which libraries where sequenced
+    // track which libraries were sequenced
     for (RunDto run : allRuns) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)